apps/trading/e2e/.env.develop

```diff
@@ -1,4 +1,4 @@
 CONSOLE_IMAGE_NAME=vegaprotocol/trading:develop
-VEGA_VERSION=v0.76.8
+VEGA_VERSION=v0.77.0-preview.3
```

Comment:
develop should stay on whatever is on mainnet. If/when we use a `next` branch we will upgrade with an .env.next